frontend/src/routes/Salary.jsx

```diff
@@ -0,0 +1,52 @@
+import { Select, Button, Typography } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const { Title } = Typography;
+
+const Salary = () => {
+  const [salaryDates, setSalaryDates] = useState([]);
+  const [selectedSalaryDate, setSelectedSalaryDate] = useState(null);
+
+  const getDate = () => {
+    axios.get(`http://localhost:3000/salary`).then((response) => {
+      const rawSalaryDateData = response.data.data;
+      const transformedSalaryDate = rawSalaryDateData.map((dateObject) =>
+        Object.values(dateObject)
+      );
+      let salaryDatesArray = [];
+
+      for (let i = 0; i < transformedSalaryDate.length; i++) {
+        salaryDatesArray[i] = { value: i, label: transformedSalaryDate[i][0] };
```

Comment:
This could also be a map, but if you're more comfortable with for loops this approach is ok too.